nixos/modules/services/backup/sanoid.nix

```diff
@@ -199,6 +199,13 @@ in
       after = [ "zfs.target" ];
       startAt = cfg.interval;
     };
+
+    # Put those packages in the global environment
+    # so that syncoid can find them when targeting this host through ssh.
+    environment.systemPackages = with pkgs; [
```

Comment:
I guess this is mostly harmless if they aren't used, but I'm not sure it makes sense to assume anyone running sanoid will also be using the machine as a syncoid target. Personally, I only add these packages for a single user on my target machines, just because it seems tidier.